ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/ILoadBalancer.java

```diff
@@ -60,8 +60,22 @@
 	
 	/**
 	 * Get the current list of servers.
-	 * 
+	 * NOTE: This method is deprecated in favor of the cleaner {@link #getReachableServers}
```

Comment:
I'm not sure why the Deprecated is not showing...weird. It's in my commit. Adding the date.